scripts/uninstall.sh

```diff
@@ -18,12 +18,22 @@ c=$(( c < 70 ? 70 : c ))
 
 PKG_MANAGER="apt-get"
 UPDATE_PKG_CACHE="${PKG_MANAGER} update"
+PKG_REMOVE="${PKG_MANAGER} -y remove --purge"
 dnsmasqConfig="/etc/dnsmasq.d/02-pivpn.conf"
 setupVarsFile="setupVars.conf"
 setupConfigDir="/etc/pivpn"
 pivpnFilesDir="/usr/local/src/pivpn"
 pivpnScriptDir="/opt/pivpn"
 
+PLAT=$(grep -sEe '^NAME\=' /etc/os-release | sed -Ee "s/NAME\=[\'\"]?([^ ]*).*/\1/")
+
+if [ "${PLAT}" == 'Alpine' ]
+then
+	PKG_MANAGER='apk'
+	UPDATE_PKG_CACHE="${PKG_MANAGER} update; ${PKG_MANAGER} upgrade --prune"
```

Comment:
`apt upgrade --prune` upgrades all packages and removes the ones which are not in repositories anymore. This is not what we want. Only the package lists/indexes are aimed to be updated, like `apt-get update` does.
```suggestion
	UPDATE_PKG_CACHE="${PKG_MANAGER} update"
```
Since the `UPDATE_PKG_CACHE` variable is now defined the same way, just contains a different value as of `PKG_MANAGER`, `UPDATE_PKG_CACHE` could be also defined after this Alpine if block and then doesn't need to be part of it anymore.